### ISSUE TITLE
CommandBlock with "/execute if entity" doesn't output the correct result with comparator

### ISSUE THREAD

**Issue by elmital** [PaperMC/Paper#2922]
### What behaviour is expected:
When you use an "/execute if entity @e" or "/execute if entity @a" in a command block with a comparator next to it the comparator output the count result.
![image](https://user-images.githubusercontent.com/54907162/73575080-a350cd00-4477-11ea-9644-b8fd715cf2a8.png)

### What behaviour is observed:
The comparator always output 1 redstone signal power if one entity is detected by the command wathever is the real number of entities and the output visible in the commandblock interface.
![image](https://user-images.githubusercontent.com/54907162/73575097-b19ee900-4477-11ea-907e-a7c38dc6b73d.png)

### Steps/models to reproduce:
- Use a commandblock with the command `/execute if entity @e` or `/execute if entity @a` 
- Next to it a comparator with a line of redstone
- Observe the output not changing whatever the entities count result

### Plugin list:
None

### Paper build number:
- Paper-35(1.15.1) 

### Anything else:
- The bug doesn't appear in Paper-34
- The bug is still present in Paper-87
- Desactivate the option use-faster-eigencraft-redstone doesn't solve the problem
- Fix in spigot maybe at the origin of the bug https://hub.spigotmc.org/jira/projects/SPIGOT/issues/SPIGOT-5502

**Comment by electronicboy** [comment]
The behavior needs comparing to vanilla, don't really know what it is there, but that looks like it's now following the intended behavior looking at the code and changes by upstream

**Comment by elmital** [comment]
After test in vanilla since the 1.13 the comparator doesn't output the count result, just 0 or 1 if the command find an entity(or much). It means that the bug fixed in spigot was present in all 1.13+ version it's why i don't have trouble using the execute command like we used before the /testfor command.

I found a report in the Mojang issue tracker : [MC-170341](https://bugs.mojang.com/browse/MC-170341?jql=text%20~%20"execute%20if%20entity")
To have the similar comportement than the "/testfor @e"(1.12) for the "execute if entity @e"(1.13+ replacement). The command is now : `execute store result block ~ ~ ~ SuccessCount int 1 if entity @e`

I close the issue, thanks for the answer :)
